README.md

```diff
@@ -1,180 +1,170 @@
+<a id="readme-top"></a>
 
-[![Tests](https://github.com/hubmapconsortium/ccf-ui/actions/workflows/tests.yml/badge.svg?branch=develop)](https://github.com/hubmapconsortium/hra-ui/actions/workflows/ci.yml)
-[![GitHub last commit](https://img.shields.io/github/last-commit/hubmapconsortium/hra-ui.svg)](https://github.com/hubmapconsortium/hra-ui/commits/develop)
-[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=hubmapconsortium_hra-ui&metric=alert_status)](https://sonarcloud.io/project/overview?id=hubmapconsortium_hra-ui)
-[![SonarCloud Coverage](https://sonarcloud.io/api/project_badges/measure?project=hubmapconsortium_hra-ui&metric=coverage)](https://sonarcloud.io/component_measures?metric=Coverage&id=hubmapconsortium_hra-ui)
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
 
-## Website Links
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
 
-- FTU User Interface (FTU)
-  - Production: <https://apps.humanatlas.io/ftu-explorer/>
-  - Staging: <https://hra-ui.netlify.app/apps/ftu-ui/>
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-## Background and More Information
+  <h3 align="center">Human Reference Atlas (HRA) User Interfaces</h3>
+</div>
 
-For more information about the Human Reference Atlas (HRA) and the Functional Tissue Unit Explorer (FTU), see the HuBMAP HRA Portal at <https://humanatlas.io>.
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
 
-## Change Log
+<!-- ABOUT THE PROJECT -->
 
-See the [ChangeLog](CHANGELOG.md) for the latest developments.
+## About The Project
 
-## Credits
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
```

Comment:
Point this to https://apps.humanatlas.io/ and upload a screenshot. You can probably ask Libby to create a good screenshot of multiple apps in a single image

Reply:
Updated the URL. Created a new issue for libby to create a image for HRA Apps